pkg/util/timeutil/timeutil.go

```diff
@@ -1,8 +1,9 @@
-package util
+package timeutil
```

Comment:
@nikovacevic @mbolt35 I don't really want to block on this, but this is once again an instance of package naming being really annoying. I think it would be awesome to name this package `time` and then use `util.time.Foo` but darn Go would make us `import timeutil "github.com/kubecost/cost-model/pkg/util/time"` instead pretty much everywhere. What do you think of, in general, this kind of package naming to avoid overloading stdlib stuff vs. aliasing imports?

Reply:
I am fine either way -- in general, I suggested the `timeutil` package because go conventions use `http` as an example. They say "don't name your package `util` and don't name it `http` (stdlib collision), but name it `httputil`" -- I, of course, agree with you, but I am just defaulting to their suggestions in the event that it somehow makes life easier. I don't know though :(

Reply:
https://blog.golang.org/package-names

"Avoid meaningless package names. Packages named util, common, or misc provide clients with no sense of what the package contains. This makes it harder for clients to use the package and makes it harder for maintainers to keep the package focused. Over time, they accumulate dependencies that can make compilation significantly and unnecessarily slower, especially in large programs. And since such package names are generic, they are more likely to collide with other packages imported by client code, forcing clients to invent names to distinguish them."

and ...

"Avoid unnecessary package name collisions. While packages in different directories may have the same name, packages that are frequently used together should have distinct names. This reduces confusion and the need for local renaming in client code. For the same reason, avoid using the same name as popular standard packages like io or http."

Reply:
I believe @mbolt35 found a Go-related resource that espoused the idea of naming packages, e.g. `httputil` instead of `util`. Which is an idea I like, for the sake of specificity. So I actually think the right move is to name the package `timeutil`, and to refactor the rest of `pkg/util` into `pkg/util/xyzutil`. What do you think about that idea?

Reply:
SGTM! I'm glad there's at least some canonical Go advice here. I wouldn't run out and refactor all of `pkg/util` but good to keep in mind for future changes like this.